framework/include/mesh/MooseMesh.h

```diff
@@ -71,7 +71,35 @@ class QpMap
 class FaceInfo
```

Comment:
We need to add a way for any active dirichlet BCs for variables to be attached to the FaceInfo object - because that is the only way we have to get those objects in to the FV variables to be used for generating ghost elem values.  This also brings a new constraint to my mind - since variables need to be initialized before materials (so materials can use them), dirichlet FV BCs won't be able to use materials - because we need to evaluate them in order to initialize the variables.  This should be okay since with FEM, we don't support material properties for dirichlet BCs - since materials don't work at nodes.

Reply:
I think I got this code in the next push.